apps/verticalhome/js/clock.js

```diff
@@ -70,6 +61,10 @@ function Clock() {
   this.handleEvent = function cl_handleEvents(e){
 
     switch(e.type) {
+      /**
+       * This event is thrown when time format has been changed.
```

Comment:
'emitted' & 'dispatch' - couple more :+1: options :)
but:  "@AlekoNN added a note a day ago

Guys, let's keep it as is please. In current circumstances we should focus only on those problems which are really major or critical. I understand that there are many minor and cosmetic issues, but we should not scatter our resources among them. The most important thing for now is to have basic functionality landed towards Harman gaia branch as soon as possible." :copyright: